packages/chain-helpers/src/blockchainApiConnection/BlockchainApiConnection.ts

```diff
@@ -24,8 +26,7 @@ export const CUSTOM_TYPES: RegistryTypes = {
   BlockNumber: 'u64',
   Index: 'u64',
   RefCount: 'u32',
-
-  ErrorCode: 'u16',
+  AccountInfo: 'AccountInfoWithProviders',
```

Comment:
As explained below, `AccountInfo` is the type used in Substrate 2.0 but with Substrate 3.0, we have to use `AccountInfoWithProviders`.